src/test/java/shop/CartTest.java

```diff
@@ -0,0 +1,75 @@
+package shop;
+
+import com.github.javafaker.Faker;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.Locale;
+
+public class CartTest {
+
+    private static String cartName;
+
+    @BeforeClass
```

Comment:
I think that @BeforeMethod is better to use here, because with current implementation 1 cart name will be used for all tests. You are creating new Cart object in every tests + don't write anything in files, so there it should work properly, but can easy break in real project, when you will add additional steps in tests. It can lead to long-time root cause analysis